client/reader/full-post/index.jsx

```diff
@@ -509,7 +513,8 @@ FullPostContainer = React.createClass( {
 	},
 
 	render: function() {
-		var passedProps = omit( this.props, [ 'postId', 'feedId' ] );
+		var passedProps = omit( this.props, [ 'postId', 'feedId' ] ),
```

Comment:
I didn't alter this line, but yeah, they are not assigned again so can be `const`. Hmm, the whole file is written in ES5, not ES6 so should I change these two lines to `const` and leave the other be?